authority/provisioner/sshpop_test.go

```diff
@@ -370,8 +372,10 @@ func TestSSHPOP_AuthorizeSSHRenew(t *testing.T) {
 			tc := tt(t)
 			if cert, err := tc.p.AuthorizeSSHRenew(context.Background(), tc.token); err != nil {
 				if assert.NotNil(t, tc.err) {
-					sc, ok := err.(render.StatusCodedError)
-					assert.Fatal(t, ok, "error does not implement StatusCodedError interface")
+					var sc render.StatusCodedError
+					if assert.True(t, errors.As(err, &sc)) {
+						assert.Equals(t, sc.StatusCode(), tc.code)
+					}
 					assert.Equals(t, sc.StatusCode(), tc.code)
```

Comment:
There are a couple of occasions of `assert.Equals(t, sc.StatusCode(), tc.code)` that seem to be duplicated in the `if` statement just before. I assume the ones outside the `if` can be removed.